### ISSUE TITLE
Command line --help --version

### ISSUE THREAD

**Issue by techtonik** [nitroshare/nitroshare-desktop#84]
For troubleshooting #65 it would be convenient to have `-h`, `--help` and `--version` arguments. Maybe `--verbose` as well.

**Comment by nathan-osman** [comment]
Although I haven't gotten around to documenting it yet, the API offers a `/version` method.

**Comment by techtonik** [comment]
It still doesn't exposed to command line.

**Comment by nathan-osman** [comment]
Here's a simple Python script you can use:

```
#!/usr/bin/env python
from json import loads
from os import path

from requests import post

with open(path.expanduser('~/.NitroShare'), 'r') as f:
    d = loads(f.read())

v = post(
    url='http://127.0.0.1:{}/version'.format(d['port']),
    json={},
    headers={'X-Auth-Token': d['token']},
)

print "NitroShare {}".format(v.json()['version'])
```

Save that to a file and make it executable. (You will need to also make sure `python-requests` is installed.)

**Comment by nathan-osman** [comment]
The next version of NitroShare will implement a proper `version` command for CLI use.

**Comment by nathan-osman** [comment]
This is implemented as of 500fa03. The command for this would be:

    nitroshare version

The results are returned as JSON:

    {
        "version": "0.4.0"
    }
